cleaned_data/modsbyPID/rftaart:8.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">RFTA_Braddock_008</identifier>
+   <identifier type="pid">rftaart:8</identifier>
+   <titleInfo>
+      <title>A family embraces their well-being</title>
+   </titleInfo>
+   <originInfo>
+      <dateIssued>2021</dateIssued>
+      <dateIssued encoding="edtf" keyDate="yes">2021</dateIssued>
+      <publisher/>
+      <place>
+         <placeTerm valueURI="http://id.loc.gov/authorities/names/n79109786">Knoxville (Tenn.)</placeTerm>
+      </place>
+   </originInfo>
+   <abstract>This is to illustrate a comment I heard in many interviews. People escaped with small things that they thought they cherished, and for those that lost so much, there was ultimately a realization that what really mattered most was the safety of loved ones . . . family and friends.</abstract>
+   <name authority="naf"
+         valueURI="http://id.loc.gov/authorities/names/n2002022941">
+      <namePart>Braddock, Paige</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/art">Artist</roleTerm>
+      </role>
+   </name>
+   <physicalDescription>
+      <form authority="aat" valueURI="http://vocab.getty.edu/aat/300033973">drawings (visual works)</form>
+   </physicalDescription>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
+      <topic>Wildfires</topic>
+   </subject>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm012404">
+      <topic>Family members</topic>
+   </subject>
+   <typeOfResource>text</typeOfResource>
+   <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+         <title>Rising from the Ashes: The Chimney Tops 2 Wildfires in Memory and Art.</title>
```

Comment:
@logantillman - Can you remove the period from the title value? Apologies that this likely was unclear in the email I sent originally.